src/config.rs

```diff
@@ -13,6 +16,66 @@ pub(crate) struct Config {
     pub(crate) epoch: SystemTime,
 }
 
+/// The kinds of networks that can be simulated in turmoil
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, Default)]
```

Comment:
Are all of these derives necessary?

Reply:
Hash may be unnessecar, and Copy is a matter of preference, but i think having 
PartialEq, Eq would be good since match!(x, y) is less intuitive than x == b